### PR TITLE
Create ListingCell for react-ui-ag

### PR DESCRIPTION
**Pull request by nray621**
[Card](https://rentpath.leankit.com/card/595782316)

- Paired with Omar Khan @okhanrt 
- Correct font and icons will be added in ag.js
- Styling in storybook matches [mocks](http://zpl.io/VkYWkJL)
- Carousel, FavoriteHeart, and Coupon are currently placeholders
- Renders one or two CTA buttons
- Passes click actions to CTA buttons, Favorite Heart, and whole ListingCell
- Clicks on CTA and Favorite Heart stopPropagation to prevent cell click from firing as well
- Takes custom RatingItem for RatingBar
- Works for single family listing

### CODE REVIEW

File: stories/ag/ListingCell.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      title: 'Cool Apartment',
+      bedroomText: '3 bedrooms',
+    }}
+    RatingItem={Star}
+    viewType="map"
+    ctaSection={[
+      {
+        children: '404-378-1428',
+        onClick: action('Phone CTA action'),
+        type: 'phone',
```
Comment (okhanrt) [inline stories/ag/ListingCell.js]: Is the type property of ctaSection ever being used? I don't see it being accessed anywhere else.

File: stories/ag/ListingCell.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      },
+      {
+        children: 'send an email',
+        onClick: action('contact CTA action'),
+        type: 'contact',
```
Comment (okhanrt) [inline stories/ag/ListingCell.js]: Ditto

File: stories/ag/ListingCell.js (hunk 1 above)
Comment (nray621) [reply stories/ag/ListingCell.js]: yeah we use it for the `data-tid`

File: stories/ag/ListingCell.js (hunk 2 above)
Comment (nray621) [reply stories/ag/ListingCell.js]: see above

File: packages/react-ui-ag/src/ListingCell/ListingCell.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    )
+  }
+
+  render() {
+    const {
+      theme,
+      viewType,
+      onCardClick,
+      listingDetails,
+      className,
```
Comment (yasabere) [inline packages/react-ui-ag/src/ListingCell/ListingCell.js]: Would it make sense to have a favoriteItem?

Comment (nray621) [reply packages/react-ui-ag/src/ListingCell/ListingCell.js]: Yeah, as long as we decide for sure that we aren't going to have a react-ui-ag favoriteHeart component and just use the react-ui-core toggleButton, then yeah. I will wait to make that change until that choice is finalized

File: stories/theme/ag/small/ListingCell.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+.ListingCell_Favorite {
+  height: 16px;
+  width: 15px;
+  position: relative;
+  bottom: 135px;
+  left: 218px;
+}
+
+.ListingCell_Info_Bottom,
```
Comment (tadjohnston) [inline stories/theme/ag/small/ListingCell.css]: You should never have 2 `_`s in a classname.  If you do, either the syntax is wrong, or your component can be broken down further.

File: stories/theme/ag/small/ListingCell.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+.ListingCell_Info_Top {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.ListingCell_Info_Top {
```
Comment (tadjohnston) [inline stories/theme/ag/small/ListingCell.css]: This is the same class as above, please combine.

**Pull request closed without merging**